src/Extensions/AzureBlobStorage/Program.cs

```diff
@@ -23,24 +23,19 @@ static async Task<int> Main(string[] args)
 
             Logger = loggerFactory.CreateLogger<Program>();
 
-            // Expected command line format is: AzureBlobStorage.exe Egress --Provider-Name MyProviderEndpointName
+            // Expected command line format is: dotnet-monitor-egress-azureblobstorage.exe Egress
             RootCommand rootCommand = new RootCommand("Egresses an artifact to Azure Storage.");
 
-            var providerNameOption = new Option<string>(
```

Comment:
See [comment](https://github.com/dotnet/dotnet-monitor/pull/2790/files#r1006173015) below for context